### PR TITLE
Add Red Hat Project Initializer

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### What does this PR do?

TODO

### CODE REVIEW

**Comment by benoitf** [comment]
Hello, FYI the ECA check is red: https://accounts.eclipse.org/legal/eca/validation/3371

**Comment by jeffmaury** [comment]
Yes but I then signed the ECA and ask for refresh but does not seems to update accordingly

**Comment by jeffmaury** [comment]
Closing as ECA check is failing, will re submit

File: plugins/redhat.project-initializer/0.0.9/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+type: VS Code extension
+name: Project Initializer
+title: Project Initializer
+description: A lightweight extension based on Red Hat launcher to generate quickstart projects using Eclipse Che
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/project-initializer/0.0.9/vspackage
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: Red Hat, Inc.
+repository: https://github.com/fabric8-launcher/launcher-vscode-extension
+category: Other
+firstPublicationDate: "2019-04-18"
```
Comment (ibuziuk) [inline plugins/redhat.project-initializer/0.0.9/meta.yaml]: No new line in the end

File: plugins/redhat.project-initializer/0.0.9/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+id: redhat.project-initializer
+version: 0.0.9
+type: VS Code extension
+name: Project Initializer
+title: Project Initializer
+description: A lightweight extension based on Red Hat launcher to generate quickstart projects using Eclipse Che
+url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/project-initializer/0.0.9/vspackage
```
Comment (ibuziuk) [inline plugins/redhat.project-initializer/0.0.9/meta.yaml]: please use GH release to avoid marketplace rate limit issue

**Pull request closed without merging**